### DISCUSSION TITLE
Cyber Chef Pivot from PCAP

### DISCUSSION THREAD

**Discussion by Cra5hedC0w** [Security-Onion-Solutions/securityonion#5735] [General] [answered]
Good day, 

I see in the release notes for 2.3.70, that there is now an option to pivot from PCAPs to cyberchef, and have those send to cyberchef ( I saw this in the video with Doug Burks on malware analysis). After upgrading to 2.3.70, I don't see this icon to pivot. Am I missing something? I've tried using recent PCAP data that was captured post update.

**Comment by Sazerak** [comment, marked as answer]
From the PCAP screen, it should be completely on the right near the top, just to the left of the Download PCAP icon. Looks like a slice of bread/toast.

**Reply by Cra5hedC0w** [reply]
I see it now. For some reason, it wasn't there when I made this post, but now is present. 

Thanks for the assist.
